lib/diagnostic.js

```diff
@@ -7,27 +7,52 @@
 // (uppercase strings without punctuation)
 // and returns that array.
 const normalizeWords = function () {
-
+  let words = text.split(/\s+/);
```

Comment:
you didn't pass an argument to your function which broke everything. it should be ```const normalizeWords = function (text)```. Otherwise this functions code looks good